go/flags/endtoend/vttablet.txt

```diff
@@ -460,6 +460,7 @@ Usage of vttablet:
       --vreplication_healthcheck_topology_refresh duration               refresh interval for re-reading the topology (default 30s)
       --vreplication_heartbeat_update_interval int                       Frequency (in seconds, default 1, max 60) at which the time_updated column of a vreplication stream when idling (default 1)
       --vreplication_max_time_to_retry_on_error duration                 stop automatically retrying when we've had consecutive failures with the same error for this long after the first occurrence
+      --vreplication_parallel_insert_workers int                         Number of parallel insertion workers to use during copy phase. Set <= 1 to disable parallelism, or > 1 enable concurrent insertion during copy phase. (default 1)
```

Comment:
We should use dashes rather than underscores in new flag names too. Sorry that I didn't note this earlier. The current plan is to move all flags to dashes in v16 so this will avoid future work.